vllm/engine/llm_engine.py

```diff
@@ -320,6 +319,13 @@ def __init__(
             if stat_loggers is not None:
                 self.stat_loggers = stat_loggers
             else:
+                # Lazy import for prometheus multiprocessing.
+                # We need to set PROMETHEUS_MULTIPROC_DIR environment variable
+                # before prometheus_client is imported.
+                # See https://prometheus.github.io/client_python/multiprocess/
+                from vllm.engine.metrics import (LoggingStatLogger,
+                                                 PrometheusStatLogger)
```

Comment:
feels like the laziness should be hidden inside `engine.metrics`

Reply:
engine.metrics imports `prometheus_client` though which we want to avoid